### ISSUE TITLE
menu name change

### ISSUE THREAD

**Issue by Theedgeclan2** [nikooo777/ckSurf#54]
this is not that big of a deal.
i use this for bhop and i dont care if its called ck or timer
but have surf is weird when people are like why you have a surf timer

**Comment by M-archand** [comment]
In ckSurf.sp line 1686 you can change what it says there and recompile.

g_hServerName = CreateConVar("ck_server_name", "**ckSurf | Surf Plugin**", "Determines the server name displayed in the timer text whilst in the start zone", FCVAR_NOTIFY);

**Comment by M-archand** [comment]
Set a cvar **ck_server_name "New name here"**

**Comment by Theedgeclan2** [comment]
so that will change !top and stuff?
i know it changes server chat name

**Comment by M-archand** [comment]
No, that will change "ckSurf | Surf Plugin" on the HUD timer in start zone.

**ck_chat_prefix** will change the chat tag from "SURF" to anything you want.

**Comment by M-archand** [comment]
Ahh sorry I see what you're saying now. If you want to change the !top menu change line 2342 in commands.sp and recompile.

**Comment by Theedgeclan2** [comment]
ok thanks. im just waiting for the speed part of the hud to be fix. its gose to far right and dont show speed
